.github/workflows/create-condalock-file.yml

```diff
@@ -0,0 +1,88 @@
+name: Conda lock file creation
+
+on:
+  push:
+    branches:
+    - main
+    - condalock
```

Comment:
I now think we should remove the triggering on pushes to main. What do you think?

Reply:
yes, totally agreed, it's more annoying than a fly around a :hankey:

Reply:
I just deactivated the test run on push, I'll do the same for Core too in a wee bit